src/main/io/osd.c

```diff
@@ -761,7 +755,7 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
 static void osdDrawLogo(int x, int y)
 {
     // display logo and help
-    char fontOffset = 160;
+    unsigned char fontOffset = 160;
```

Comment:
`uint8_t` is used in CF, but it may be better to use `int` and wrap explicitly
```
if (fontOffset < 255) {
                 displayWriteChar(osdDisplayPort, x + column, y + row, fontOffset++);
}
```
BTW: is really 37% of available characters used for logo?